src/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { userInformationProps } from "./types";
+import { errorsProps } from "./types";
+
+export const checkValues = (userInformation: userInformationProps): errorsProps => {
+    let errors = {
+        flagForuserNameErrors: false, userNameError: "",
```

Comment:
keep them separated , each one in separate line that will be much obvious to read.